scripts/image-names.sh

```diff
@@ -17,7 +17,11 @@ bashVarPrefix() {
 
 # These names must match the images specified in Helm configs
 prodImageName() {
-  echo "vro-$1"
+  if [ "$1" == "rabbitmq" ]; then
```

Comment:
can you comment as to why we have this special case

Reply:
oh i meant in the code 😅  just bc when someone comes back to this they'll think, why?

Reply:
Ahh true.

Reply:
Done. 

For next steps, which github action runs this script? and where can I confirm this change has occurred? @msnwatson

Reply:
so, you'll have to run this script manually to generate the right output but then you'll also have to change the deployment pipelines for prod to use this new name and then also change the deployments in the argo repo to use this new image name pattern

Reply:
be very cautious as this is a breaking change to our deployment of rabbitmq and there is a lot of stuff going on with that image right now. communicate this to the team probably in slack

Reply:
Thanks for the head up. I will do this tomorrow when things slow down.